subprojects/docs/src/docs/release/notes.md

```diff
@@ -1,15 +1,24 @@
-
 The Gradle team is excited to announce Gradle @version@.
 
-This is the third patch release for Gradle 7.6.
+This is the fourth patch release for Gradle 7.6.
+
+This release addresses security vulnerabilities:
+
+* [Apache Ant insecure temporary file vulnerability](https://github.com/gradle/gradle/security/advisories/GHSA-j45w-qrgf-25vm)
+* [Guava insecure temporary directory](https://github.com/gradle/gradle/issues/25945)
+* [Ivy Improper Restriction of XML External Entity Reference](https://nvd.nist.gov/vuln/detail/CVE-2022-46751) and related Gradle issue [#27855](https://github.com/gradle/gradle/issues/24795)
```

Comment:
❌ The second link label and issue do not match: 27855 vs 24795

Reply:
Thanks! No idea where it came from.